backend/src/database/migrations/20221009141317-change-attachments-metadata-to-jsonb-array.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.changeColumn(
```

Comment:
Are we sure we can just do this without taking care of the existing rows?

Reply:
yup it works — tested on local and will test on staging

haha I wonder if this PR is necessary — now that I think about it, I don't really think we are running operations on this column now. it's just that in the process of researching this, I saw somewhere that says a `json[]` is almost always an error 🤔

(relatedly, wonder if a similar change should be made to the other columns like `params` in `email_messages` and `email_messages_transactional`)

Reply:
hmm if that's the case, let's update everything if we're to update. I'd rather have bad but consistent code than inconsistently  good code

Reply:
OK I will update everything then